client/src/app/routes.tsx

```diff
@@ -93,6 +94,10 @@ const buildAppRoutes = (
         commonRoutes.push(
             <Route path="/:network/currency-converter" component={CurrencyConverter} key={keys.next().value} />
         );
+    } else {
+        commonRoutes.push(
+            <Route path="/:network/statistics" component={Statistic} key={keys.next().value} />
```

Comment:
This needs to be on condition that it's a "stardust" network.

Reply:
Done.